dnf.spec

```diff
@@ -21,7 +21,7 @@ URL:		https://github.com/rpm-software-management/dnf
 # cd dnf/package
 # ./archive
 # tarball will be generated in $HOME/rpmbuild/sources/
-Source0:    http://rpm-software-management.fedorapeople.org/dnf-%{version}.tar.gz
+Source0:    https://github.com/rpm-software-management/dnf/archive/%{name}-%{version}-1.tar.gz
 BuildArch:  noarch
```

Comment:
but if you are tagging a release like just you tagged https://github.com/rpm-software-management/dnf/releases/tag/dnf-1.1.1-2 this then why not upload the source auto generated by tagging? I see following works fine
wget https://github.com/rpm-software-management/dnf/archive/dnf-1.1.1-2.tar.gz